wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Anchor.kt

```diff
@@ -245,4 +243,73 @@ class Anchor(
         .transactions
     }
   }
+
+  /**
+   * Get account transactions for specified asset. Optional field implementation depends on anchor.
+   *
+   * @param assetCode asset's code
+   * @param authToken auth token of the account authenticated with the anchor
+   * @param toml Anchor's stellar.toml file containing `CURRENCIES` list of supported assets
+   * @param limit optional how many transactions to fetch
+   * @param pagingId optional return transactions prior to this ID
+   * @param noOlderThan optional return transactions starting on or after this date and time
+   * @param lang optional language code specified using
+   * [RFC 4646](https://www.rfc-editor.org/rfc/rfc4646), default is `en`
+   *
+   * @return a list of formatted operations
+   *
+   * @throws [NetworkRequestFailedException] if network request fails
+   * @throws [AssetNotSupported] if asset is not supported by the anchor
+   */
+  suspend fun getHistory(
```

Comment:
Keeping both `getHistory` and `getAllTransactionStatus` for now, but we can probably remove or repurpose `getAllTransactionStatus`.